gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/util/EhrResourceExtractorUtil.java

```diff
@@ -63,11 +63,11 @@ private static boolean validObservationStatement(RCMRMT030101UK04Component4 comp
         return component.getObservationStatement() != null && component.getObservationStatement().getId() == resourceId;
     }
 
-    private static boolean hasEhrComposition(RCMRMT030101UK04Component3 component) {
+    public static boolean hasEhrComposition(RCMRMT030101UK04Component3 component) {
```

Comment:
You can move this and `hasEhrFolder` method higher, so they're over private methods.